<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBarang</class>
 <widget class="QWidget" name="FormBarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idBarangLabel">
      <property name="text">
       <string>Id barang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idBarangLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaBarangLabel">
      <property name="text">
       <string>Nama barang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaBarangLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="satuanLabel">
      <property name="text">
       <string>Satuan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="satuanLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hargaLabel">
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="hargaLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="keteranganLabel">
      <property name="text">
       <string>Keterangan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="keteranganLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="beratLabel">
      <property name="text">
       <string>Berat</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="beratLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="dilihatLabel">
      <property name="text">
       <string>Dilihat</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="dilihatLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="dibeliLabel">
      <property name="text">
       <string>Dibeli</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="dibeliLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="kataKunciLabel">
      <property name="text">
       <string>Kata Kunci</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="kataKunciLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="stokLabel">
      <property name="text">
       <string>Stok</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="stokLineEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="idSubKategoriLabel">
      <property name="text">
       <string>Id sub kategori</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="idSubKategoriLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableBarang">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
